ufs_cmds.c

```diff
@@ -806,6 +806,9 @@ static int do_conf_desc(int fd, __u8 opt, __u8 index, char *data_file)
 		}
 		lseek(data_fd, 0, SEEK_SET);
 
+		if (file_size > QUERY_DESC_CONFIGURAION_MAX_SIZE)
+			file_size = QUERY_DESC_CONFIGURAION_MAX_SIZE;
+
```

Comment:
In case an user create the wrong config file. I don't think the tool shall try fix it. Better if the user will receive Invalid length Query error and fix it.